### PR TITLE
Remove {{css_key_concepts}}

### PR DESCRIPTION
**Pull request by teoli2003**
This is the last occurrence on mdn/content.

It will unblock mdn/yari#6819

### CODE REVIEW

**Comment by SphinxKnight** [comment]
thanks @teoli2003 and mea culpa for missing this one

**Comment by estelle** [comment]
There are two quotes in links, which look wrong to me, but not sure what you were trying to do.

File: files/en-us/web/css/visual_formatting_model/index.md [hunk 1]
```diff
@@ -142,4 +142,12 @@ A block box is a block-level box that is also a block container. As described in
 
 ## See also
 
-- {{css_key_concepts}}
+- CSS Key Concepts: [CSS syntax](/en-US/docs/Web/CSS/Syntax), [at-rule](/en-US/docs/Web/CSS/At-rule"), [comments](/en-US/docs/Web/CSS/Comments),
```
Comment (estelle) [inline files/en-us/web/css/visual_formatting_model/index.md:145]:
```suggestion
- CSS Key Concepts: [CSS syntax](/en-US/docs/Web/CSS/Syntax), [at-rule](/en-US/docs/Web/CSS/At-rule), [comments](/en-US/docs/Web/CSS/Comments),
```

File: files/en-us/web/css/visual_formatting_model/index.md [hunk 2]
```diff
@@ -142,4 +142,12 @@ A block box is a block-level box that is also a block container. As described in
...
+  [specificity](/en-US/docs/Web/CSS/Specificity") and [inheritance](/en-US/docs/Web/CSS/inheritance"),
```
Comment (estelle) [inline files/en-us/web/css/visual_formatting_model/index.md:146]:
```suggestion
  [specificity](/en-US/docs/Web/CSS/Specificity) and [inheritance](/en-US/docs/Web/CSS/inheritance),
```

**Review by estelle — APPROVED** [review]

**Pull request merged**